CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 CHANGELOG
 =========
 
+* 0.16.2 (2017-06-14)
```

Comment:
can you remove the date (: i am nor sure if this will be relased today. we will update this when doing! use `dev-master` as headline